### PR TITLE
IGNITE-16386 Set clusterId on node restart

### PR DESCRIPTION
**Pull request by nizhikov**

### CODE REVIEW

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/conflictresolve/CacheConflictResolutionManagerImpl.java [hunk 1]
```diff
@@ -38,22 +39,42 @@
...
 
     /** {@inheritDoc} */
     @Override public CacheVersionConflictResolver conflictResolver() {
+        cctx.logger(this.getClass()).info("Conflict resolver created [" +
+            "cache=" + cctx.name() +
+            ", clusterId=" +  clusterId +
+            ", conflictResolveField=" + conflictResolveField + ']'
+        );
+
+        IgniteLogger log = cctx.logger(CacheVersionConflictResolverImpl.class);
```
Comment (ololo3000) [inline modules/cdc-ext/src/main/java/org/apache/ignite/cdc/conflictresolve/CacheConflictResolutionManagerImpl.java:64]: cctx.logger() creates new logger instance each time it is called. Do we need it? May be it is better to save it once in the dedicated variable?

File: modules/cdc-ext/src/test/java/org/apache/ignite/cdc/CacheConflictOperationsTest.java [hunk 2]
```diff
@@ -41,6 +41,7 @@
 import org.apache.ignite.internal.processors.cache.KeyCacheObjectImpl;
 import org.apache.ignite.internal.processors.cache.dr.GridCacheDrInfo;
 import org.apache.ignite.internal.processors.cache.version.GridCacheVersion;
+import org.apache.ignite.testframework.GridTestUtils;
```
Comment (ololo3000) [inline modules/cdc-ext/src/test/java/org/apache/ignite/cdc/CacheConflictOperationsTest.java:44]: Unused import.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/conflictresolve/CacheVersionConflictResolverPluginProvider.java [hunk 3]
```diff
@@ -123,9 +135,16 @@ public void setConflictResolveField(String conflictResolveField) {
...
 
+    /** @param name Plugin name. */
+    public void setName(String name) {
+        this.name = name;
+    }
+
     /** {@inheritDoc} */
     @Override public void start(PluginContext ctx) {
-        // No-op.
+        IgniteEx ign = (IgniteEx)ctx.grid();
```
Comment (ololo3000) [inline modules/cdc-ext/src/main/java/org/apache/ignite/cdc/conflictresolve/CacheVersionConflictResolverPluginProvider.java:145]: It seems that that we don't need ctx variable anymore.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/conflictresolve/CacheConflictResolutionManagerImpl.java (hunk 1 above)
Comment (nizhikov) [reply modules/cdc-ext/src/main/java/org/apache/ignite/cdc/conflictresolve/CacheConflictResolutionManagerImpl.java:64]: Fixed.

File: modules/cdc-ext/src/test/java/org/apache/ignite/cdc/CacheConflictOperationsTest.java (hunk 2 above)
Comment (nizhikov) [reply modules/cdc-ext/src/test/java/org/apache/ignite/cdc/CacheConflictOperationsTest.java:44]: Fixed

**Review by shishkovilja — APPROVED** [review]

**Review by ololo3000 — APPROVED** [review]

**Pull request merged**